tailcfg/tailcfg.go

```diff
@@ -1596,3 +1604,18 @@ type SSHAction struct {
 	// value.
 	HoldAndDelegate string `json:"holdAndDelegate,omitempty"`
 }
+
+// ServerMetaResponse is the JSON response to a request to /meta?v=<n>
+// over HTTPS (regular TLS) to the Tailscale control plane server,
+// where <n> is the client's CurrentMapRequestVersion.
+type ServerMetaResponse struct {
+	// PublicNacl specifies the server's original NaCl-based machine key.
+	// It will be zero for sufficiently new clients, based on their
+	// advertised "v" parameter (the CurrentMapRequestVersion).
+	// In that case, only the Noise-based transport may be used.
+	PublicKeyNacl key.MachinePublic `json:"publicKeyNacl"`
```

Comment:
Want to add the fields for key rotation?

Reply:
Done.